src/lib.rs

```diff
@@ -144,142 +160,6 @@ impl Sub<Timespec> for Timespec {
     }
 }
 
-/**
- * Returns the current time as a `timespec` containing the seconds and
- * nanoseconds since 1970-01-01T00:00:00Z.
- */
-pub fn get_time() -> Timespec {
-    let (sec, nsec) = sys::get_time();
-    Timespec::new(sec, nsec)
-}
-
-
-/**
- * Returns the current value of a high-resolution performance counter
- * in nanoseconds since an unspecified epoch.
- */
-pub fn precise_time_ns() -> u64 {
-    sys::get_precise_ns()
```

Comment:
It depends on the underlying OS(or bare metal stack or whatever) though and thus can't be provided as part of the no-std library impl.